README.md

```diff
@@ -26,20 +39,20 @@ Reference implementation of Conventional Commits specification.
                 |  <summary> <newline> <newline> <body>
                 |  <summary>
 
-<summary>      ::= <type> "(" <scope> ")" ":" <summary-text>
-                |  <type> ":" <summary-text>
+<summary>      ::= <type> "(" <scope> ")" ":" *<whitespace> <text>
+                |  <type> ":" *<whitespace> <text>
 <type>         ::= 1*<any UTF8-octets except newline or parens or ":" or whitespace>
 <scope>        ::= 1*<any UTF8-octets except newline or parens>
-<summary-text> ::= 1*<any UTF8-octets except newline>
+<text>         ::= 1*<any UTF8-octets except newline>
 
 <footer>       ::= <token> ":" <value>
 <token>        ::= <type> "(" <scope> ")"
                 |  <type>
                 |  "BREAKING CHANGE"
-<value>        ::= <summary-text> 1*<continuation>
-                |  <summary-text>
-<continuation> ::= <newline> <whitespace> <summary-text>
+<value>        ::= <text> 1*<continuation>
+                |  <text>
+<continuation> ::= <newline> <whitespace> <text>
 
 <body>         ::= 1*<body-text>
-<body-text>    ::= <newline>? <summary-text>
+<body-text>    ::= <newline>? <text>
```

Comment:
This isn't quite right, as we'd like to allow `\n\n` in message bodies, I  think perhaps we should allow `\n\n` unless

The next token matches a `<footer>`.